### PR TITLE
[Refactor]refactor log workload group when query arrives be

### PR DESCRIPTION
**Pull request by wangbo**
### What problem does this PR solve?

Issue Number: close #xxx

Related PR: #xxx

Problem Summary:
Add wg info when query arrives BE.

1. with group
```
I20241205 14:43:46.636260 4101563 fragment_mgr.cpp:670] query_id: df11980d08ca493d-8d60f0ff07980a40, coord_addr: TNetworkAddress(hostname=127.0.0.1, port=9220), total fragment num on c
urrent host: 8, fe process uuid: 1733370035373, query type: SELECT, report audit fe:TNetworkAddress(hostname=127.0.0.1, port=9220), use wg:TG[id = 1, name = normal, cpu_share = 20, mem
ory_limit = 169.12 GB, enable_memory_overcommit = true, version = 13, cpu_hard_limit = 1, scan_thread_num = 16, max_remote_scan_thread_num = 960, min_remote_scan_thread_num = 8, memory_
low_watermark=50, memory_high_watermark=80, is_shutdown=false, query_num=0, read_bytes_per_second=-1, remote_read_bytes_per_second=-1]
```

2. without group
```
I20241205 14:44:18.009614 4101564 fragment_mgr.cpp:670] query_id: 2e963fb1b323486f-b55bbbe898ba248c, coord_addr: TNetworkAddress(hostname=127.0.0.1, port=9220), total fragment num on current host: 8, fe process uuid: 1733370035373, query type: SELECT, report audit fe:TNetworkAddress(hostname=127.0.0.1, port=9220), use wg:not carry wg
```

### Release note

None

### Check List (For Author)

- Test 
    - [ ] Regression test
    - [ ] Unit Test
    - [x] Manual test (add detailed scripts or steps below)
    - [ ] No need to test or manual test. Explain why:
        - [ ] This is a refactor/code format and no logic has been changed.
        - [ ] Previous test can cover this change.
        - [ ] No code files have been changed.
        - [ ] Other reason 

- Behavior changed:
    - [x] No.
    - [ ] Yes. 

- Does this need documentation?
    - [x] No.
    - [ ] Yes. 

### Check List (For Reviewer who merge this PR)

- [ ] Confirm the release note
- [ ] Confirm test cases
- [ ] Confirm document
- [ ] Add branch pick label

### CODE REVIEW

**Comment by wangbo** [comment]
run buildall

File: be/src/runtime/fragment_mgr.cpp [hunk 1]
```diff
@@ -653,12 +653,26 @@ Status FragmentMgr::_get_or_create_query_ctx(const TPipelineFragmentParams& para
             }
 
             if (!query_ctx) {
+                WorkloadGroupPtr workload_group_ptr = nullptr;
+                std::string wg_info_str = "not carry wg";
```
Comment (yiguolei) [inline be/src/runtime/fragment_mgr.cpp:657]: 这种写法太chinese了
Workload group not set

File: be/src/runtime/fragment_mgr.cpp [hunk 2]
```diff
@@ -653,12 +653,26 @@ Status FragmentMgr::_get_or_create_query_ctx(const TPipelineFragmentParams& para
...
+                if (params.__isset.workload_groups && !params.workload_groups.empty()) {
+                    uint64_t wg_id = params.workload_groups[0].id;
+                    workload_group_ptr =
+                            _exec_env->workload_group_mgr()->get_task_group_by_id(wg_id);
```
Comment (yiguolei) [inline be/src/runtime/fragment_mgr.cpp:661]: rename  this method get_task_group_by_id to get_group  
not using task group
not add id

File: be/src/runtime/fragment_mgr.cpp [hunk 3]
```diff
@@ -653,12 +653,26 @@ Status FragmentMgr::_get_or_create_query_ctx(const TPipelineFragmentParams& para
...
+                    if (workload_group_ptr != nullptr) {
+                        wg_info_str = workload_group_ptr->debug_string();
+                    } else {
+                        wg_info_str = "carrying wg but not find it in be";
```
Comment (yiguolei) [inline be/src/runtime/fragment_mgr.cpp:665]: And  here, too chinese

**Comment by wangbo** [comment]
run buildall

**Review by yiguolei — APPROVED** [review]
LGTM

**Review by xinyiZzz — APPROVED** [review]
LGTM

**Pull request merged**